niworkflows/data/__init__.py

```diff
@@ -1,182 +1,5 @@
-"""Niworkflows data files
-
-.. autofunction:: load
-
-.. automethod:: load.readable
-
-.. automethod:: load.as_path
-
-.. automethod:: load.cached
```

Comment:
Please keep this section, as it populates the docs: https://www.nipreps.org/niworkflows/master/api/niworkflows.data.html